recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
```

Comment:
If it's a recipe on conda-forge as a feedstock, by definition there should be builds for it.  These builds are at:

https://anaconda.org/conda-forge/icu/56.1/download/linux-64/icu-56.1-2.tar.bz2

When you run conda-build, you can specify additional channels, or you can just add conda-forge to your .condarc.

    conda build qt5 -c conda-forge